src/promptflow/promptflow/_sdk/_service/templates/ui_traces.html

```diff
@@ -14,6 +14,7 @@
     </style>
```

Comment:
TODO: make this HTML auto refresh - may not do in this PR

Reply:
Yes, Let's do it in future PR.